### PR TITLE
Use seven as base theme

### PR DESCRIPTION
**Pull request by ol0lll**

### CODE REVIEW

File: templates/README.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ Please add your changes accordingly.
 
 Template file | Usage | Solution
 ------------- | ------------- | ------------- 
-./admin-block-content.html.twig | Identical to seven. | -
-./block--local-actions-block.html.twig | Identical to seven. | -
-./block-content-add-list.html.twig | Identical to seven. | -
+./admin-block-content.html.twig | Identical to seven. | Removed.
```
Comment (dbosen) [inline templates/README.md]: Can't we just delete the lines of removed files?

File: thunder_admin.theme [hunk 2]
```diff
@@ -485,3 +388,13 @@ function _thunder_admin_remove_contextual_links(array &$variables) {
     $variables['attributes']['class'] = array_diff($variables['attributes']['class'], ['contextual-region']);
   }
 }
+
+/**
+ * Implements hook_library_info_alter().
+ */
+function thunder_admin_library_info_alter(&$libraries, $extension) {
+  if ($extension == 'core') {
+    // Re-add stable vertical-tabs for happy screenshotting @todo REMOVE.
```
Comment (dbosen) [inline thunder_admin.theme]: small typo "screenshotting"

File: thunder_admin.theme [hunk 3]
```diff
@@ -308,6 +206,11 @@ function thunder_admin_form_node_form_alter(&$form, FormStateInterface $form_sta
       '#markup' => '<h4 class="label inline">' . t('Author') . '</h4> ' . $author_info,
     ],
   ];
+
+  // Revert change in seven: Display rev info below meta and remove details.
+  // @todo REMOVE.
```
Comment (dbosen) [inline thunder_admin.theme]: When this is revoed, tha whole thunder_admin_form_node_form_alter() can be reduced to contain _thunder_admin_content_form_alter_helper() only. The other form alters in this function are part of drupal core nowadays.

File: thunder_admin.theme (hunk 2 above)
Comment (ol0lll) [reply thunder_admin.theme]: https://en.wiktionary.org/wiki/screenshotting
https://ell.stackexchange.com/questions/126789/screenshoting-or-screenshotting

Comment (dbosen) [reply thunder_admin.theme]: OK :)

**Pull request merged**